### ISSUE TITLE
Ansible release candidate comparison throws error when running role

### ISSUE THREAD

**Issue by mattjamesaus** [DataDog/ansible-datadog#445]
Get the following error when running the role on the following verson of ansible, it would seem it's related to the RC version of ansible? Is the intention to block it running on RCs?

```
TASK [datadog.datadog : Fail early if Python 3 is used on CentOS / RHEL < 8 with old Ansible] ***********************************************************************
"msg": "The conditional check '(not datadog_ignore_old_centos_python3_error) and (ansible_version.full is version(\"2.11\", operator=\"lt\", version_type='semver')) and (ansible_pkg_mgr == \"yum\") and (ansible_facts.python.version.major | int >= 3)' failed. The error was: Version comparison failed: invalid semantic version '2.13.2rc1'\n\nThe error appears to be in '/********************/roles/datadog.datadog/tasks/pkg-redhat.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Fail early if Python 3 is used on CentOS / RHEL < 8 with old Ansible\n  ^ here\n"}
```
```
ansible [core 2.13.2rc1]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/matt/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/matt/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.4 (main, Apr  2 2022, 09:04:19) [GCC 11.2.0]
  jinja version = 3.0.3
  libyaml = True
➜  ~ 
```

**Comment by bkabrda** [comment]
Hi 👋 thanks for the report. It's certainly not our intention to block our playbook from running on RCs. I think the ansible [version test](https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions) that is used here doesn't handle RC releases properly and that makes the task fail. I will need to look into this closer and figure out how to fix it.

**Comment by bkabrda** [comment]
Fixed in #446, the fix will be part of the next release. Thanks again for the report!
